### ISSUE TITLE
Wrong path names in win builds

### ISSUE THREAD

**Issue by almaz-khan** [laynezh/vite-plugin-lib-assets#17]
Hello, 

When I'm using the plugin on my Windows machine, I'm getting wrong path names in `css` files:

`./assets\a0719bd606186699.ttf`

I was able to fix that by changing `emitFile` return to 
```
  return assetPath.replace('\\', '/');
```

Not sure if that brakes anything else, it works for my case. I can create PR if that is ok solution. Or maybe there are easier way to fix that, but I haven't found anything in docs.

**Comment by coder-layne** [comment]
Try v0.5.0, it worked correctly on my Windows PC.

**Comment by almaz-khan** [comment]
Thank you a lot, it worked. I have another issue with paths like:  `../../assets/fonts/font-file.ttf` . I'm getting an issue: `file not found '../assets/fonts/font-file.ttf'`. However with the files with path `./assets/fonts/font-file.ttf` it works like a charm

**Comment by coder-layne** [comment]
Can you provide a minimal reproduction example?

**Comment by almaz-khan** [comment]
Sorry for delay. Here is the repository https://github.com/almaz-khan/vite-lib . I'm getting
```
[commonjs--resolver] Could not set source for asset "assets/[contenthash].svg", asset sour
ce needs to be a string, Uint8Array or Buffer.
error during build:
RollupError: Could not set source for asset "assets/[contenthash].svg", asset source needs
 to be a string, Uint8Array or Buffer.
    at error (file:///C:/Users/almaz-khan/workspace/vite-lib/node_modules/rollup/dist/es/
shared/node-entry.js:2245:30)
    at getValidSource (file:///C:/Users/almaz-khan/workspace/vite-lib/node_modules/rollup
/dist/es/shared/node-entry.js:24880:16)
    at FileEmitter.emitAsset (file:///C:/Users/almaz-khan/workspace/vite-lib/node_modules
/rollup/dist/es/shared/node-entry.js:25040:15)
    at FileEmitter.emitFile (file:///C:/Users/almaz-khan/workspace/vite-lib/node_modules/
rollup/dist/es/shared/node-entry.js:24920:25)
    at emitFile (file:///C:/Users/almaz-khan/workspace/vite-lib/node_modules/@laynezh/vit
e-plugin-lib-assets/dist/index.js:53:13)
    at file:///C:/Users/almaz-khan/workspace/vite-lib/node_modules/@laynezh/vite-plugin-l
ib-assets/dist/index.js:138:29
    at Array.forEach (<anonymous>)
    at Object.resolveId (file:///C:/Users/almaz-khan/workspace/vite-lib/node_modules/@lay
nezh/vite-plugin-lib-assets/dist/index.js:137:21)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

**Comment by coder-layne** [comment]
Thank you once again for your feedback. This problem has been fixed in v0.5.1.
